test/acme/data/HibernateAdapterTest.java

```diff
@@ -115,7 +115,7 @@ public void testCustomer() {
 	
 	@Test
 	public void testTicket() {
-		Ticket ticket = new Ticket();
+		Ticket ticket = new Ticket(null);

```

Comment:
This looks bad. The default constructor for `Ticket` should have a call to `this(null)` if necessary.

Reply:
Can't because of hibernate